src/assets/translations/fr/main.json

```diff
@@ -97,9 +102,12 @@
     "gem": "GEM",
     "gemMessage": "Acheter ou Vendre des cryptos avec Gem",
     "junoPay": "JunoPay",
+    "junoPayMessage": "Achetez des cryptos avec JunoPay",
     "banxa": "Banxa",
     "banxaMessage": "Achetez ou vendez des cryptos avec Banxa",
     "comingSoon": "Bientôt disponible ...",
+    "coinbasePay": "Coinbase Pay",
+    "coinbasePayMessage": "Acheter des cryptos  avec Coinbase Pay",
```

Comment:
Additional whitespace? 
```suggestion
    "coinbasePayMessage": "Acheter des cryptos avec Coinbase Pay",
```

Reply:
@firebomb1 This one is correct in Gitlocalize, it's just an additional whitespace that got in the PR